api/handler/attributes.go

```diff
@@ -0,0 +1,231 @@
+package handler
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/nspcc-dev/neofs-s3-gw/api"
+	"github.com/nspcc-dev/neofs-s3-gw/api/data"
+	"github.com/nspcc-dev/neofs-s3-gw/api/errors"
+	"github.com/nspcc-dev/neofs-s3-gw/api/layer"
+)
+
+type (
+	GetObjectAttributesResponse struct {
+		ETag         string       `xml:"ETag,omitempty"`
+		ObjectSize   int64        `xml:"ObjectSize,omitempty"`
+		StorageClass string       `xml:"StorageClass,omitempty"`
+		ObjectParts  *ObjectParts `xml:"ObjectParts,omitempty"`
+	}
+
+	ObjectParts struct {
+		IsTruncated          bool   `xml:"IsTruncated,omitempty"`
+		MaxParts             int    `xml:"MaxParts,omitempty"`
+		NextPartNumberMarker int    `xml:"NextPartNumberMarker,omitempty"`
+		PartNumberMarker     int    `xml:"PartNumberMarker,omitempty"`
+		Parts                []Part `xml:"Part,omitempty"`
+		PartsCount           int    `xml:"PartsCount,omitempty"`
+	}
+
+	Part struct {
+		PartNumber int `xml:"PartNumber,omitempty"`
+		Size       int `xml:"Size,omitempty"`
+	}
+
+	GetObjectAttributesArgs struct {
+		MaxParts         int
+		PartNumberMarker int
+		Attributes       []string
+		VersionID        string
+	}
+)
+
+const (
+	maxPartsDefault         = 1000 // limit number of parts on a response
```

Comment:
I've not found in docs what value should the variable have, that's why it's just a some number.

Reply:
Use `layer.MaxSizePartsList`.